package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "resin-semver",
   "version": "0.1.1",
-  "description": "resin specific semver utility methods",
+  "description": "Resin.io specific semver utility methods",
   "main": "src/index.js",
   "typings": "src/index.d.ts",
   "scripts": {
     "test": "mocha -r ts-node/register --outDir tmp test/index.spec.ts",
-    "build": "tsc src/index.ts"
+    "build": "tsc src/index.ts",
+    "ci": "npm run test"
   },
   "author": "Lucian Buzzo <lucian@resin.io>",
   "license": "Apache-2.0",
   "dependencies": {
-    "lodash.memoize": "^4.1.2",
-    "semver": "^5.3.0"
+    "lodash.memoize": "4.1.2",
+    "semver": "5.3.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.0.1",
-    "@types/lodash.memoize": "^4.1.2",
-    "@types/mocha": "^2.2.41",
-    "@types/semver": "^5.3.32",
-    "chai": "^4.1.0",
-    "mocha": "^3.4.2",
-    "ts-node": "^3.2.1",
-    "typescript": "^2.4.2"
+    "@types/chai": "4.0.1",
+    "@types/lodash.memoize": "4.1.2",
+    "@types/mocha": "2.2.41",
+    "@types/semver": "5.3.32",
+    "chai": "4.1.0",
+    "mocha": "3.4.2",
+    "ts-node": "3.2.1",
+    "typescript": "2.4.2"
```

Comment:
(`typescript` is one exception I think you can probably get away with here, since it's pretty major and I know we've had specific problems there, but for other packages I think this holds)